biztalk/adapters-and-accelerators/lob-and-enterprise-2020-support.md

```diff
@@ -0,0 +1,145 @@
+---
+# required metadata
+
+title: BizTalk Server 2020: Supported Line-of-Business (LOB) and Enterprise systems | Microsoft Docs
+description: Details of Line-of-Business (LOB) and Enterprise systems supported by BizTalk Server 2020
+author: msjaydeep
+ms.author: jaah
+manager: dougeby
+ms.date: 01/03/2019
+ms.topic: support-metrics
+ms.prod: biztalk-server
+
+# optional metadata
+
+#ROBOTS:
+
+ms.reviewer: 
+ms.suite:
+ms.tgt_pltfrm:
+ms.assetid: 
+ms.custom: biztalk-2020
+---
+# BizTalk Server 2020: Supported Line-of-Business (LOB) and Enterprise systems
+
+## Introduction 
+
+This article lists the Line-of-Business (LOB) and Enterprise systems supported by `BizTalk Server 2020`. These details are about adapters installed with `BizTalk Server 2020` and `BizTalk Adapters for Enterprise Applications`
+
+> [!NOTE]
+> There is no seperate installation for BizTalk Adaper Pack (BAP). These are installed with BizTalk Server 2020 release onwards. 
```

Comment:
Applicable starting BizTalk Server 2020, BizTalk Adapter Pack (BAP) is included in BizTalk Server installation and requires no separate installation.